samza-core/src/test/java/org/apache/samza/container/grouper/task/TestGroupByContainerCount.java

```diff
@@ -244,7 +243,7 @@ public void testBalancerAfterContainerIncrease() {
    *  from C2 and C3 to containers that were on the same respective hosts, it wasn't implemented
    *  because the scenario is infrequent, the benefits are not guaranteed, and the code complexity
    *  wasn't worth it. It certainly could be implemented in the future.
-   */
+   *//*
```

Comment:
Ah.. these bunch of tests. I think I forgot to get to them after fixing the other million test failures. Let me work on this. Thanks for pointing out Yi! :)